docs/integrations.md

```diff
@@ -156,3 +156,25 @@ In addition, ruff publishes the following images:
 
 As with the distroless image, each image is published with ruff version tags as
 `ruff:{major}.{minor}.{patch}-{base}` and `ruff:{major}.{minor}-{base}`, e.g., `ruff:0.6.6-alpine`.
+
+## GitLab CI/CD
+
+You can add the following configuration to `.gitlab-ci.yml` to run a `ruff format`, and a `ruff check` compatible with GitLab's codequality.
+
+```yaml
+Ruff:
+  stage: .pre
```

Comment:
Should we use the `build` stage to ensure the job runs even if it is the only job?

> If a pipeline contains only jobs in the .pre or .post stages, it does not run. There must be at least one other job in a different stage. ([source](https://docs.gitlab.com/ee/ci/yaml/index.html#stage-pre))

Reply:
Yeah fair, not a great idea to assume _anything_ about the end-users setups hah